osu.Game/Overlays/Login/LoginPanel.cs

```diff
@@ -68,6 +68,14 @@ private void load()
             apiState.BindValueChanged(onlineStateChanged, true);
         }
 
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            userStatus.BindTo(api.LocalUser.Value.Status);
+            userStatus.BindValueChanged(e => updateDropdownCurrent(e.NewValue), true);
```

Comment:
This is not correct though now is it?

`api.LocalUser.Value.Status` is a bindable field _under_ a bindable. `api.LocalUser` will change value after e.g. a log-out and a log-in. Upon which `userStatus` will not be bound to `api.LocalUser.Value.Status` anymore, it will be bound to the _old_ user's `Status` bindable.

There was a reason this was placed where it was placed.

Reply:
Yep, you got that one correct. Pretty annoying usage of bindables here, will adjust.